src/app/child-dev-project/notes/notes-of-child/notes-of-child.component.ts

```diff
@@ -49,9 +51,10 @@ export class NotesOfChildComponent
       visibleFrom: "md",
     },
     {
-      name: "author",
+      name: "authors",
       label: "SW",
-      inputType: ColumnDescriptionInputType.TEXT,
+      inputType: ColumnDescriptionInputType.READONLY,
+      asyncValueFunction: (note: Note) => this.getAuthorNames(note.authors),
```

Comment:
This approach is rather ineffective, for every entry, all the users have to be loaded and can than only be displayed. A faster approach would be to just load all of them once and then filter out the once needed in the value function.
However, the entity subrecord component is getting refactored and also this problem will be solved in #764.